.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I thought the .vscode folder was gitignored? We should kill this file I think.

Reply:
I've removed it.